src/main/java/org/cactoos/io/ReaderAsBytes.java

```diff
@@ -135,6 +135,7 @@
             }
             builder.append(buffer, 0, done);
         }
+        this.reader.close();
```

Comment:
@llorllale What does this close() have to do with the task of creating class TempFile?